Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1355,6 +1356,37 @@ static inline double normalizedFlexFraction(const GridTrack& track)
     return track.baseSize() / std::max<double>(1, flexFactor);
 }
 
+void IndefiniteSizeStrategy::accumulateFlexFractionMasonry(double& flexFraction, unsigned trackIndex, GridTrackSizingDirection outermostDirection, GridLayoutState& gridLayoutState) const
+{
+    auto& allTracks = m_algorithm.tracks(outermostDirection);
+    auto trackLength = allTracks.size();
+    const Grid& grid = m_algorithm.grid();
+
+    for (auto track = 0u; track < trackLength; track++) {
```

Comment:
`track` should be called `trackIndex`

Reply:
We already have trackIndex defined above, so that does not work.